pkgs/os-specific/linux/kernel/linux-hardkernel-4.14.nix

```diff
@@ -0,0 +1,17 @@
+{ stdenv, buildPackages, hostPlatform, fetchurl, perl, buildLinux, libelf, utillinux, ... } @ args:
+
+buildLinux (args // rec {
+  version = "4.14.47-139";
+
+  # modDirVersion needs to be x.y.z.
+  modDirVersion = "4.14.47";
+
+  # branchVersion needs to be x.y. 
+  extraMeta.branch = "4.14";
+
+  src = fetchurl {
```

Comment:
This should use `fetchFromGitHub` to avoid possibly nondeterministic GitHub tarballs.

Looks good otherwise.

Reply:
Ugh, is nothing sacred!! I assumed GitHub's tarballs at least would be deterministic.

Will fix. Thanks for looking!